sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <Description>Azure Client SDK integration with Microsoft.Extensions libraries</Description>
     <AssemblyTitle>Azure Client SDK integration Microsoft.Extensions</AssemblyTitle>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>1.1.0-preview.1</VersionPrefix>
```

Comment:
We probably need to do a pass over the repo and change any of these `VersionPrefix` properties to `Version`

Reply:
I'll do that